src/main/java/mcp/mobius/betterbarrels/client/render/TileEntityBarrelRenderer.java

```diff
@@ -39,14 +39,16 @@ public void renderTileEntityAt(TileEntity tileEntity, double xpos, double ypos,
 			// the following is how it was set, enabling lighting makes it look better with smooth lighting, mostly... bottom labels are abit dark
 			// will need to actually properly calculate lighting, but for now this looks better on smooth lit barrels than the full bright it was before
 			// found another bug: disabling lighting causes grass to look white, and a few other quirks, putting this on hold for the momment
-			int[][] savedGLState = modifyGLState(new int[]{ GL11.GL_BLEND, GL11.GL_LIGHTING }, null);
+            GL11.glPushAttrib(GL11.GL_BLEND | GL11.GL_LIGHTING);
```

Comment:
I don't think that will work as described here:
https://registry.khronos.org/OpenGL-Refpages/gl2.1/xhtml/glPushAttrib.xml
I suggest to change it to GL_ENABLE_BIT, it will save both of this attributes.
```suggestion
            GL11.glPushAttrib(GL11.GL_ENABLE_BIT);
```

Reply:
Oups yeah, i just copied them from the method before.